Formula/volta.rb

```diff
@@ -0,0 +1,27 @@
+class Volta < Formula
+  desc "Volta: The JavaScript Launcher"
+  homepage "https://volta.sh"
+  url "https://github.com/volta-cli/volta.git", :revision => "10a9bee148d2a833069d6cc27aea56af3092dc6e"
+  version "0.7.2"
+
+  depends_on "rust" => :build
+
+  conflicts_with "hadoop", :because => "both install `yarn` binaries"
+  conflicts_with "node", :because => "both install `node` binaries"
+  conflicts_with "npm", :because => "both install `npm` binaries"
+  conflicts_with "yarn", :because => "both install `yarn` binaries"
```

Comment:
Why does it install these ? Can it depend on Homebrew's `node`, `npm`, `yarn` instead? If it cannot: `keg_only` seems like a much better idea than four `conflicts_with`.

Reply:
It shouldn’t actually be a conflict, though users may end up with oddities if using both and their PATH isn’t configured correctly. These conflicts should be removed.

Context: Volta is a toolchain Manager for Node, comparable to (though substantially different in design from) things like nvm, nodenv, etc.

Reply:
Where are `yarn` etc. installed?

Reply:
Effectively (there are symlinks involved) in a path (usually in users' home directories, but configurable) that Volta manages, normally `~/.volta/bin`, with appropriate tweaks to add that to the  users' `PATH`.

Reply:
I wouldn't add any of these conflicts, then, unless any of these symlinks are installed into `/usr/local/bin` by default.

Reply:
Entirely agreed—that's exactly what I meant when noting that these are not conflicts.

Reply:
Resolved on d142d754e.